### ISSUE TITLE
`from_input` method should load taskrc_location and data_location

### ISSUE THREAD

**Issue by lyz-code** [robgolding/tasklib#58]
Hi, I'm writing some hooks and I noticed that the `from_input` method loads a TaskWarrior backend with the following configuration

```python
backend = backends.TaskWarrior(data_location=hook_parent_dir)
```

Therefore it loads the data from the hook parent directory but the configuration from `~/.taskrc`. 

Now with the hooks V2 it would be as easy as to load those two values from `sys.argv`.

I'm willing to do a Pull Request if you are interested.

Please let me know

**Comment by xerus2000** [comment]
yes, please implement that :)

**Comment by lyz-code** [comment]
Sorry @xerus2000, due to the lack of activity of taskwarrior/tasklib I've migrated my efforts to [pydo](https://github.com/lyz-code/pydo)

**Comment by xerus2000** [comment]
Hmpf, that's sad. I have been working myself into taskwarrior now. I don't wanna switch again, but I'm already hitting pain points.

I do understand that the representation as text file isn't always optimal, but it is the UNIX philosophy and can be really useful if you work a lot with text-processing CLI tools - renaming tags or whatever can easily be done without even using taskwarrior.

**Comment by lyz-code** [comment]
I agree with you, but [these reasons](https://lyz-code.github.io/pydo/#why-another-cli-task-manager) made me take the decision. Regarding the UNIX point, I think that as long as the application supports the import and export of it's data as json, it makes it compliant with the principle.
